watson/cli.py

```diff
@@ -127,8 +127,10 @@ def wrapper(*args, **kwargs):
 
 @click.group(cls=DYMGroup)
 @click.version_option(version=_watson.__version__, prog_name='Watson')
+@click.option('--color/--no-color', 'color', default=None,
+              help="(Don't) show colored output.")
```

Comment:
I think we can find a better description for this. Maybe "Force the colored output to be activated (--color) or disabled (--no-color)"?

Reply:
Sorry about the delay. I think indeed, that's a better description, but this file uses this help convention, examples:

```
help="(Don't) Stop an already running project."
help="(Don't) include currently running frame in report."
help="(Don't) view output through a pager."
```

Applicable suggestion:

```suggestion
              help="Force the colored output to be activated (--color) or disabled (--no-color).")
```

Reply:
@jmaupetit : any opinion on this?

Reply:
Maybe the following to have a better description and respect currently used pattern? :sweat_smile: 
```suggestion
              help="(Don't) force colored output to be activated.")
```

Reply:
@jmaupetit: I realized it may be confusing. Using `--no-color` would expand to: `Don't force colored output to be activated`, but the action is different: `Force colored output to be deactivated`. 

After some thought I think @k4nar suggestion seems better than original one and the one above.

Possible other solutions:
* Remove `--no-color` completely and use just a `--color` flag (like in the other PR). 
* Other phrasings:
  * Forces output to be colored or non-colored.
  * Force colored output to be activated (--no-color for deactivation).

Reply:
I think it's useful to be able to bypass custom configuration, and thus keep the `--no-color` flag. Final proposition:

```suggestion
              help="(Don't) color output.")
```